Lib/fontTools/subset/__init__.py

```diff
@@ -2776,6 +2790,7 @@ def _closure_glyphs(self, font):
 					log.info("Closed glyph list over '%s': %d glyphs after",
 							 table, len(self.glyphs))
 					log.glyphs(self.glyphs, font=font)
+			setattr(self, f"glyphs_{table.lower()}ed", frozenset(self.glyphs))
```

Comment:
Nit: it appears we unnecessarily write glyphs_bslned if bsln [what the heck is a bsln] exists?

Reply:
[`bsln`](https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6bsln.html) is an Apple AAT table. It's true that attribute is unused, but same for `self.glyphs_cmaped` or `self.glyphs_glyfed`. I'd just keep it there, it's a pattern. Can be useful for debugging.